### PR TITLE
Add v0.9-alpha release notes

### PR DESCRIPTION
**Pull request by lukekim**

### CODE REVIEW

**Review by ewgenius — DISMISSED** [review]

**Review by sgrebnov — DISMISSED** [review]

File: docs/release_notes/v0.9-alpha.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Spice v0.9-alpha (Mar 14, 2024)
+
+The v0.9 release adds several data connectors including the Spice data connector for the ability to connect to other `spiced` instances. Improved observability for `spiced` has been added with the new `/metrics` endpoint for monitoring deployed instances.
```
Comment (mitchdevenport) [inline docs/release_notes/v0.9-alpha.md:3]: Technically support to connect to other `spiced` instances already existed

File: docs/release_notes/v0.9-alpha.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Highlights in v0.9-alpha
+
+**Spice.ai data connector**: Use other `spiced` instances as data connectors for federated SQL query across `spiced` deployments and for chaining `spiced` deployments.
+
+**Keyring secret store**: Use the operating system native credential store, like macOS keychain for storing secrets used by `spiced`.
+
+**PostgreSQL data connector**: [PostgreSQL](https://www.postgresql.org/) can now be used as both a data store for acceleration and as a connector for federated SQL query.
```
Comment (mitchdevenport) [inline docs/release_notes/v0.9-alpha.md:13]: Do we want to mention Data Mesh support too?

File: docs/release_notes/v0.9-alpha.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+**Databricks data connector**: [Databricks](https://www.databricks.com) as a connector for federated SQL query across Delta Lake tables.
+
+**Metrics endpoint**: Added new `/metrics` endpoint for `spiced` observability and monitoring.
+
```
Comment (mitchdevenport) [inline docs/release_notes/v0.9-alpha.md:32]: Is it worth calling out s3 support too since our goal is to show we support as much as possible?

**Review by digadeesh — DISMISSED** [review]

File: docs/release_notes/v0.9-alpha.md (hunk 2 above)
Comment (lukekim) [reply docs/release_notes/v0.9-alpha.md:13]: We'll use the term federated SQL query for now.

File: docs/release_notes/v0.9-alpha.md (hunk 3 above)
Comment (lukekim) [reply docs/release_notes/v0.9-alpha.md:32]: Done!

**Review by sgrebnov — DISMISSED** [review]

**Review by digadeesh — DISMISSED** [review]

File: docs/release_notes/v0.9-alpha.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- @y-f-u
+- @digadeesh
+- @lukekim
+
+## New in this release
+
+- **Adds** Keyring secret store (`keyring`).
+- **Adds** PostgreSQL data connector (`postgres`).
+- **Adds** Spice.ai data connector (`spiceai`).
+- **Adds** Arrow Flight SQL (JDBC) support.
```
Comment (phillipleblanc) [inline docs/release_notes/v0.9-alpha.md]:
```suggestion
- **Adds** Arrow Flight SQL (JDBC/ODBC/ADBC) support.
```

File: docs/release_notes/v0.9-alpha.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+# Spice v0.9-alpha (Mar 14, 2024)
+
+The v0.9 release adds several data connectors including the Spice data connector for the ability to connect to other `spiced` instances. Improved observability for `spiced` has been added with the new `/metrics` endpoint for monitoring deployed instances.
+
+## Highlights in v0.9-alpha
```
Comment (phillipleblanc) [inline docs/release_notes/v0.9-alpha.md:5]: Should we add Flight SQL server as a highlight?

Comment (lukekim) [reply docs/release_notes/v0.9-alpha.md:5]: Done

**Review by ewgenius — APPROVED** [review]

**Review by phillipleblanc — APPROVED** [review]

**Pull request merged**